account_wallet/wizards/wizard_account_move_refund_wallet.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2022 ACSONE SA/NV (http://www.acsone.eu)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import _, fields, models
+
+
+class AccountMoveRefund(models.TransientModel):
+
+    _name = "wizard.account_move_refund.wallet"
+    _description = "Refund By Wallet"
+
+    account_wallet_type_id = fields.Many2one(
+        comodel_name="account.wallet.type",
+        string="Wallet type",
+        required=True,
```

Comment:
```suggestion
        required=True,
        ondelete='cascade'
```

Same under for required fields